src/snowflake/cli/plugins/spcs/services/manager.py

```diff
@@ -24,7 +28,7 @@ def create(
         spec = self._read_yaml(spec_path)
 
         query = f"""\
-            CREATE SERVICE IF NOT EXISTS {service_name}
+            CREATE SERVICE {service_name}
             IN COMPUTE POOL {compute_pool}
```

Comment:
I would consider this a bug fix, since without specifically specifying "--if-not-exists", the intended result of trying to create a service with the same name as an existing one should be an error, not success.

Reply:
make sense